### DISCUSSION TITLE
release for M1?

### DISCUSSION THREAD

**Discussion by eltonooi** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#101] [General]
can't wait to have it ! ;)
or do have any alternative yet?

**Comment by davidwernhart** [comment]
Currently working hard on the new M1 version and making some decent progress, stay tuned :)

**Comment by MatthiasKerbl** [comment]
The time has come: AlDente for Apple Silicon (M1 MacBooks) is out now. We thank everyone who has helped us here in the forum and hope you like it. That's not all, we have also started working on a pro version of AlDente with advanced features several weeks ago. Since we have put a lot of hours into developing the app, it will cost a little. However, there will always be this free and open-source version of AlDente updated with the new M1 SMC keys.  If you have any feature requests or ideas for AlDente Pro, let us know. If you want to get notified, when AlDente Pro comes out, just enter your email at https://apphousekitchen.com/aldente/ In the meantime, stay safe and keep your battery fresh. David & Matthias
